ports/mimxrt/boards/MAKERDIARY_RT1011_NANO_KIT/board.json

```diff
@@ -0,0 +1,19 @@
+{
+    "deploy": [
+        "deploy_makerdiary.md"
+    ],
+    "docs": "",
+    "features": [
+        "External Flash",
+        "USB",
+        "microSD"
+    ],
+    "images": [
+        "MAKERDIARY_RT1011_NANO_KIT.jpg"
+    ],
+    "mcu": "mimxrt",
+    "product": "Makerdiary iMX RT1011 Nano Kit",
```

Comment:
Does the product need to include the word "Makerdiary"?  See #16031.

Reply:
No. Makerdiary does not include it on their web site.

Reply:
I changed the name in the deploy instructions too.